common/testing/testtelemetry/collector.go

```diff
@@ -0,0 +1,72 @@
+package testtelemetry
```

Comment:
Nothing changed here except I renamed the previous `OTELLogger` to `MemoryCollector`.